src/irmin-graphql/unix/irmin_graphql_unix.ml

```diff
@@ -25,7 +25,8 @@ module Server = struct
   end
 
   module Make_ext
-      (S : Irmin.Generic_key.S) (Remote : sig
+      (S : Irmin.Generic_key.S)
+      (Remote : sig
         val remote : remote_fn option
       end)
```

Comment:
This is great!

Nitpick questions: It looks like the`Remote : sig...end` could fit on a single line? The `end` is missing an indentation space to indicate that it is within the opening parens?

Reply:
Indenting the `end` one more means indenting the body of the `sig` one more too ? This would cause a huge diff.

Indenting the `end` but not the body is not so bad:

```ocaml
  module Make_ext
      (S : Irmin.Generic_key.S)
      (Remote : sig
        val remote : remote_fn option
       end)
```

I would like the opinion of more people on this :)

Reply:
Nested things are indented 2 + width of the parens, [for example in a `if`](https://github.com/mirage/irmin/pull/2262/files#diff-2c45f709798cc1744a16c0854299ffade4a111e4325068f3afea417f1188c8a7R235). 

I'm not against indenting the `end` but without a strong opinion, I'm tempted to leave it like that.

Reply:
Would it be more consistent with the "2 space + width of paren" rule to indent this like the following by adding 1 space (width of paren) to both `val` and `end` -- essentially format them relative to `Remote` instead of `(Remote`? The current formatting does not seem to take the paren into consideration (unlike the diff I linked in my prior comment), but maybe in this context of functor arguments the intention is for the opening paren to be considered the first column of the block (?). 

```ocaml
  module Make_ext
      (S : Irmin.Generic_key.S)
      (Remote : sig
         val remote : remote_fn option
       end)
```